actors/miner/src/emit.rs

```diff
@@ -0,0 +1,27 @@
+// A namespace for helpers that build and emit Miner Actor events.
+
+use fil_actors_runtime::runtime::Runtime;
+use fil_actors_runtime::{ActorError, EventBuilder};
+use fvm_shared::sector::SectorID;
+
+/// Indicates a sector has been pre-committed.
+pub fn sector_precommited(rt: &impl Runtime, id: SectorID) -> Result<(), ActorError> {
```

Comment:
I know the SectorID type has exactly what you want, but please pass two separate parameters here for the miner actor ID and sector number separately. The SectorID type only exists as part of SealVerifyInfo; it's defined outside of our repo and is unused except when forced by the FVM SDK API. I was genuinely surprised to see it as I'd forgotten it exists.

Reply:
This is done.